sources/packages/backend/apps/api/src/route-controllers/report/report.aest.controller.ts

```diff
@@ -25,11 +25,12 @@ import {
 import BaseController from "../BaseController";
 import { ReportsFilterAPIInDTO } from "./models/report.dto";
 import { FormNames } from "../../services/form/constants";
+import { Role } from "../../auth/roles.enum";
 import {
-  REPORT_CONFIG_NOT_FOUND,
   FILTER_PARAMS_MISMATCH,
-} from "../../services/report/constants";
-import { Role } from "../../auth/roles.enum";
+  ReportService,
+  REPORT_CONFIG_NOT_FOUND,
```

Comment:
I would say that `REPORT_CONFIG_NOT_FOUND` must come from `@sims/services/constants` and ReportService must be in scope of API with exclusively for methods used in API. I see here the method is shared by API and integration. Hence IMO it must be in `@sims/services`